src/Microsoft.DocAsCode.Dfm/FileCacheLite.cs

```diff
@@ -121,7 +113,24 @@ public FileCacheModel()
 
             private FileStream CreateTempFile()
             {
-                return new FileStream(Path.GetTempFileName(), FileMode.Create, FileAccess.ReadWrite, FileShare.None, 4096, FileOptions.DeleteOnClose | FileOptions.SequentialScan);
+                const int MaxRetry = 3;
+                int retry = 0;
```

Comment:
>retry [](start = 20, length = 5)

we have RetryIO in FileWriterBase and IncrementalUtility, is it possible to put it into one utility?